classes/Currency.php

```diff
@@ -1126,7 +1126,7 @@ public static function countActiveCurrencies($idShop = null)
             self::$countActiveCurrencies[$idShop] = Db::getInstance()->getValue('
 				SELECT COUNT(DISTINCT c.id_currency) FROM `' . _DB_PREFIX_ . 'currency` c
 				LEFT JOIN ' . _DB_PREFIX_ . 'currency_shop cs ON (cs.id_currency = c.id_currency AND cs.id_shop = ' . (int) $idShop . ')
-				WHERE c.`active` = 1
+				WHERE c.`active` = 1 AND c.`deleted` = 0
```

Comment:
To be consistent with function name, I suggest to add a new optional bool params like `excludeDeleted` ?

Reply:
Why not, but this PR will not be testable anymore and I should make another one in the module.

What do you think about it?

Reply:
Function name is `countActiveCurrencies`, I think this parameter should not be needed.